docs/src/whatsnew/3.0.rst

```diff
@@ -6,8 +6,7 @@ v3.0 (25 Jan 2021)
 This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
-
-.. dropdown:: :opticon:`report` Release Highlights
+.. dropdown:: :opticon:`report` v3.0.0 Release Highlights
```

Comment:
@pp-mo To mirror this on `v3.1.x`, also see #4322

Reply:
:+1: